main.cpp

```diff
@@ -65,11 +79,56 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+   
+    // ====================================== 
+    // WingTangWong's entries 
+    // ====================================== 
     
-    
-    
+    // Integers
+    int    numberOfCars       = 32;
+    int    numberOfSpareTires = 8192;
+    int    numberOfBolts      = 16384;
+
+    // Character
+    char   letterOfLastName  = 'w';
+    char   letterOfGender    = 'm';
+    char   letterOfFailGrade = 'f';
+
+    // Boolean
+    bool   isItemBurning     = true;
+    bool   isItemWet         = false;
+    bool   isItemFull        = false;
+
+    // Floating Point
+    float  realPiValue     = 3.14159f;
+    float  realRepeatValue = 1.18181f;   // (13.0/11.0) = 1.18181818181818181818
+    float  realPercentage  = 0.15867f;  // (1234/7777) = 0.15867301015815867301
+
+    // Double Floating Point
+    double longerPiValue     = 3.14159265358979323; 
+    double longerRepeatValue = 1.18181818181818181; // (  13.0/  11.0)
+    double longerPercentage  = 0.15867301015815867; // (1234.0/7777.0)
+
+    // Ignore them since they are unused
+    ignoreUnused(numberOfCars);
+    ignoreUnused(numberOfSpareTires);
+    ignoreUnused(numberOfBolts);
+    ignoreUnused(letterOfLastName);
+    ignoreUnused(letterOfGender);
+    ignoreUnused(letterOfFailGrade);
+    ignoreUnused(isItemBurning);
+    ignoreUnused(isItemWet);
+    ignoreUnused(isItemFull);
+    ignoreUnused(realPiValue);
+    ignoreUnused(realRepeatValue);
+    ignoreUnused(realPercentage);
+    ignoreUnused(longerPiValue);
+    ignoreUnused(longerRepeatValue);
+    ignoreUnused(longerPercentage);
```

Comment:
- [ ] pass them all to a single call of `ignoreUnused()`, like in the `rentACar()` implementation.

Reply:
Done. Collapsed into single call.